Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -292,6 +292,70 @@ void CFrame::OpenGeneralConfiguration(wxWindowID tab_id)
   m_main_config_dialog->SetFocus();
 }
 
+void CFrame::CycleProfile(bool forward) {
+  //load the hotkey config
+  InputConfig* hk_conf = HotkeyManagerEmu::GetConfig();
+  auto hotkey_controller = hk_conf->GetController(0);
+
+  //load the wiimote config
+  InputConfig* wm_conf = Wiimote::GetConfig();
+  std::string pname(File::GetUserPath(D_CONFIG_IDX));
+
+  //find all profiles
+  pname += "Profiles/" + wm_conf->GetProfileName();
+  std::vector<std::string> sv = Common::DoFileSearch({ pname }, { ".ini" });
+
+  //remove all profiles that do not fit the filter string
+  if (!hotkey_controller->profile_filter.empty()){
```

Comment:
Brace goes on the next line in our coding style.

Reply:
Fixed.